docs/diagnostics/PX1077.md

```diff
@@ -0,0 +1,87 @@
+# PX1077
+This document describes the PX1077 diagnostic.
```

Comment:
@EAndrosova please review the documentation for PX1077 diagnostic

Reply:
The edited file is attached
[PX1077.md](https://github.com/user-attachments/files/17199196/PX1077.md)